pkg/imgutil/snapshotter_test.go

```diff
@@ -132,6 +136,12 @@ func TestRemoteSnapshotterOpts(t *testing.T) {
 				checkRemoteSnapshotterAnnotataions, checkStargzSnapshotterAnnotataions,
 			},
 		},
+		{
+			name: "soci",
+			check: []func(t *testing.T, a map[string]string){
+				checkRemoteSnapshotterAnnotataions,
+			},
```

Comment:
Do we need checkSociSnapshotterAnnotataions? Any annotations can be checked in SOCI?